### ISSUE TITLE
request to add categorical labels for multi_class classification but not in numerical categories.

### ISSUE THREAD

**Issue by sathishkumar369** [pycaret/pycaret#262]
Hi pycaret team!

when I gave prediction on unseen data, the output come up with label values.
I want it in categorical(original form) form but not in label form.

hope you will understand my points by looking into images.

Actual inputs mentioned in the figure and it's a multiclass one

![Screenshot (244)](https://user-images.githubusercontent.com/40235632/86370484-70686a80-bc9d-11ea-9535-8373fadd0c85.png)

predicted one mentioned in the below figure

![Screenshot (242)](https://user-images.githubusercontent.com/40235632/86370528-7d855980-bc9d-11ea-9444-4744d2952364.png)

**Comment by pycaret** [comment]
Thanks for the suggestion @sathishkumar369.

Tagging this as an enhancement.

**Comment by ingebert** [comment]
Hi Team,
thanks for making pycaret so useful and awesome, and thanks Satish for the suggestion.
I would highly appreciate this feature, too!
If I understand this right, this would be an implementation of sklearn.preprocessing.LabelEncoder.inverse_transform(transformed_label_as_in_prediciton)
It looks like the "sklearn label encoder" is already used in pycaret code but could be made accessible by some option in setup? I´m not good enough in coding (yet :) ) and hope this can be done easily or with workaround by making the encoded/decoded array accessible (see picture).
![image](https://user-images.githubusercontent.com/68894423/89284087-7fe92380-d64e-11ea-89a5-581eb27d3f85.png)
I tried already to access it with get_config('prep_pipe') but couldn´t find it.

**Comment by Yard1** [comment]
I added a PR for this, https://github.com/pycaret/pycaret/pull/348

**Comment by pycaret** [comment]
This issue is resolved. 

Thanks for working on this @Yard1
